modules/configuration/src/main/java/org/apache/ignite/internal/configuration/tree/InnerNode.java

```diff
@@ -202,12 +205,29 @@ public abstract void construct(
     @Override
     public InnerNode copy() {
         try {
-            return (InnerNode) clone();
+            InnerNode clone = (InnerNode) clone();
+
+            clone.immutable = false;
+
+            return clone;
         } catch (CloneNotSupportedException e) {
             throw new IllegalStateException(e);
         }
     }
 
+    public final void assertMutability() {
```

Comment:
Missing javaDoc

Reply:
Could you add a line before each method call to make it easier to read.

Reply:
What exactly do you mean here? Could you please show me an example where the line is missing?

Reply:
Leave a comment with an example.

Reply:
Example of what?